hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/metadata/DBTransactionBuffer.java

```diff
@@ -32,5 +33,5 @@ <KEY, VALUE> void addToBuffer(Table<KEY, VALUE> table, KEY key, VALUE value)
   <KEY, VALUE> void removeFromBuffer(Table<KEY, VALUE> table, KEY key)
       throws RocksDatabaseException, CodecException;
 
-  void close() throws RocksDatabaseException;
+  void close() throws IOException;
```

Comment:
Why revert #9482 in `DBTransactionBuffer`?  The patch compiles fine without this change.

Reply:
Let me make this change